src/draw/lv_draw_label.c

```diff
@@ -398,29 +399,36 @@ static void draw_letter(lv_draw_unit_t * draw_unit, lv_draw_glyph_dsc_t * dsc,
         return;
     }
 
-    uint32_t bitmap_size = lv_draw_buf_width_to_stride(g.box_w, LV_COLOR_FORMAT_A8) * g.box_h;
-    /*Round up to avoid many allocations if the next buffer is just slightly larger*/
-    bitmap_size = LV_ALIGN_UP(bitmap_size, 64);
-    if(dsc->_bitmap_buf_size < bitmap_size) {
-        lv_draw_buf_free(dsc->_bitmap_buf_unaligned);
-        dsc->_bitmap_buf_unaligned = lv_draw_buf_malloc(bitmap_size, LV_COLOR_FORMAT_A8);
-        LV_ASSERT_MALLOC(dsc->_bitmap_buf_unaligned);
-        dsc->bitmap_buf = lv_draw_buf_align(dsc->_bitmap_buf_unaligned, LV_COLOR_FORMAT_A8);
-        dsc->_bitmap_buf_size = bitmap_size;
+    lv_draw_buf_t * draw_buf = lv_draw_buf_reshape(dsc->_draw_buf, 0, g.box_w, g.box_h, 0);
```

Comment:
In case of vector fonts, the `lv_font_get_glyph_bitmap` parameter of `lv_font_get_glyph_bitmap` is always ignored because the return value is used, right? 

In case of image fonts it can return either a pointer to an `lv_image_dsc_t` or a path. So the `draw_buf` is not used here either.

If so, can we have the allocation only if `if(g.bpp == LV_DRAW_LETTER_GLYPH_FORMAT_A8)`?

Reply:
Done